src/tox.ini

```diff
@@ -15,7 +15,7 @@ deps =
 commands = pytest {posargs}
 
 [testenv:cert]
-envlist = py38,py39,py310
+envlist = py39,py310,py311
```

Comment:
Make a corresponding change in `.github/workflows/github-actions-test.yml` to add Python 3.11 to the automated test suite